src/components/Movie/MovieBanner.js

```diff
@@ -0,0 +1,31 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+const MovieBanner = (props) => {

```

Comment:
You can use the destructuring to not repeting all the time `props.movie.genres`.
Another thing, is you're only using the data inside movie, so, pass this object direct as a prop, not inside an object, this create an unnecessary layer.

Reply:
Destructuring was implemented.